back/boxtribute_server/app.py

```diff
@@ -20,3 +24,25 @@ def configure_app(app, *blueprints, database_interface=None, **mysql_kwargs):
 
     app.config["DATABASE"] = database_interface or create_db_interface(**mysql_kwargs)
     db.init_app(app)
+
+
+def main(*blueprints):
+    """Integrate Sentry SDK. Create and configure Flask app."""
+    # dsn/environment/release: reading SENTRY_* environment variables set in CircleCI
+    sentry_sdk.init(
+        integrations=[FlaskIntegration()],
+        traces_sample_rate=float(os.getenv("SENTRY_TRACES_SAMPLE_RATE", 1.0)),
```

Comment:
What is this for?

Reply:
Cf. https://docs.sentry.io/platforms/python/configuration/sampling/#setting-a-uniform-sample-rate